3ds/3_cleanup.sh

```diff
@@ -3,6 +3,6 @@
 ################################################################
 # Cleanup library build folders and other stuff
 
-rm -rf zlib-*/ freetype-*/ harfbuzz-*/ icu/ icu-native/ libmad-*/ libmodplug-*/ libogg-*/ libpng-*/ libvorbis-*/ pixman-*/ mpg123-*/ libsndfile-*/ speexdsp-*/ sdl-wii/ tremor-lowmem/ .patches-applied
-rm -rf ctrulib/ sf2dlib/ lpp-3ds_libraries/
+rm -rf zlib-*/ freetype-*/ harfbuzz-*/ icu/ icu-native/ libmad-*/ libmodplug-*/ libogg-*/ libpng-*/ libvorbis-*/ pixman-*/ mpg123-*/ libsndfile-*/ speexdsp-*/ sdl-wii/ tremor-lowmem/ wildmidi-*/ .patches-applied
```

Comment:
`sdl-wii` should be removed here (but since `mad`, `modplug` and `ogg` are gone soon, this does not really matter).